src/protocol.rs

```diff
@@ -325,7 +325,11 @@ pub mod close {
     pub type MerchantSendAuthorization = Session! {
         // Tezos authorization signature
         recv MutualCloseAuthorizationSignature;
-        // Merchant verifies the signature
+        CustomerVerifySignature;
+    };
+
+    pub type CustomerVerifySignature = Session! {
+        // Customer verifies the signature
```

Comment:
On review, I think this separation can be removed since I no longer need the type name for a function definition. Is it a useful name anyway?

Reply:
removed.